processing/src/main/java/io/druid/query/filter/DimFilterCacheHelper.java

```diff
@@ -37,7 +37,7 @@
   static final byte JAVASCRIPT_CACHE_ID = 0x7;
   static final byte SPATIAL_CACHE_ID = 0x8;
   static final byte IN_CACHE_ID = 0x9;
-  static final byte STRING_SEPARATOR = (byte) 0xFF;
+  public static final byte STRING_SEPARATOR = (byte) 0xFF;
```

Comment:
I'm not sure I understand this comment.  Are we requesting that all things be made public because some of the things are now used from other packages?  Or is the request to just redo the things that are being made public here in the package where they are being used?